README.md

```diff
@@ -36,4 +36,9 @@ Running tests:
 - `make test`
 
 Running benchmarks:
-- `make bench`
\ No newline at end of file
+- `make bench`
+
+## Running in a container
+- `docker build . -t graphql_server`
+- `docker run --rm -p 3030:3030 -d graphql_server`
```

Comment:
I personally would not include `--rm` in the default `docker run` command but I guess that's a matter of preference.